src/qilib/configuration_helper/adapters/d5a_instrument_adapter.py

```diff
@@ -62,8 +65,8 @@ def apply(self, config: PythonJsonStructure) -> None:
         self._instrument.set_dac_unit(unit)
         dac_parameters = {param: values for param, values in config.items() if param[0:3] == 'dac'}
         for dac, values in dac_parameters.items():
-            self._instrument[dac].step = values['step']
-            self._instrument[dac].inter_delay = values['inter_delay']
+            cast(Parameter, self._instrument[dac]).step = values['step']
+            cast(Parameter, self._instrument[dac]).inter_delay = values['inter_delay']
```

Comment:
As you can see, an assignment IS done to 2 members of Parameter.

The result type of __getitem__ in InstrumentBase is Union[Callable, Parameter].
step and inter_delay aren't members off Callable, so I casted to make this explicit that we are only talking about Parameters here and not Callable. Because mypy is allowing it, I will remove this cast.